HF-122: Framework integration guides for React, Angular, Vue, Svelte (#1653)
handsontable/hyperformula: docs/guide/integration-with-react.md

# Integration with React

> The HyperFormula API is identical in a React app and in plain JavaScript. What changes is where the engine lives in a component tree and how its lifecycle maps to React hooks.

Reviewer: ```suggestion
The HyperFormula API is identical in a React app and in plain JavaScript. This guide demonstrates how HyperFormula is integrated with the React component tree and how its lifecycle maps to React hooks.
```

Author: Adopted verbatim in [`b080681`](https://github.com/handsontable/hyperformula/pull/1653/commits/b080681dc). Better wording — thanks.
